docs/_docs/best-practices/general/general.md

```diff
@@ -28,6 +29,157 @@ Discover essential FinOps best practices to optimize cost efficiency and governa
 
 ---
 
+## Resource inventory summary
+
+### Query: Count of all resources
+
+This Azure Resource Graph (ARG) query counts all resources within the specified scope.
+
+<h4>Category</h4>
+
+Inventory
+
+<h4>Query</h4>
+
+```kql
+Resources
+| summarize count()
+```
+
+<br>
+
+### Query: Count of all resources per subscription
```

Comment:
This still feels too simplistic, right? If we think this and the previous one are needed, should we merge both with the next one, which is closer to what I think people are looking for with a resource inventory?